.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
could you include `/.vscode/` in `.gitignore` file ? so that this file will not be included